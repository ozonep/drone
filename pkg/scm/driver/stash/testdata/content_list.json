{
  "size": 85,
  "limit": 25,
  "isLastPage": false,
  "values": [
    "content.go",
    "content_test.go",
    "git.go",
    "git_test.go",
    "issue.go",
    "issue_test.go",
    "linker.go",
    "linker_test.go",
    "org.go",
    "org_test.go",
    "pr.go",
    "pr_test.go",
    "repo.go",
    "repo_test.go",
    "review.go",
    "review_test.go",
    "stash.go",
    "stash_test.go",
    "testdata/branch.json",
    "testdata/branch.json.golden",
    "testdata/branches.json",
    "testdata/branches.json.golden",
    "testdata/changes.json",
    "testdata/changes.json.golden",
    "testdata/commit.json"
  ],
  "start": 0,
  "nextPageStart": 25
}
