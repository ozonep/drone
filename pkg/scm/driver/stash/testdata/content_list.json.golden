[
  {
    "path": "content.go",
    "kind": "file"
  },
  {
    "path": "content_test.go",
    "kind": "file"
  },
  {
    "path": "git.go",
    "kind": "file"
  },
  {
    "path": "git_test.go",
    "kind": "file"
  },
  {
    "path": "issue.go",
    "kind": "file"
  },
  {
    "path": "issue_test.go",
    "kind": "file"
  },
  {
    "path": "linker.go",
    "kind": "file"
  },
  {
    "path": "linker_test.go",
    "kind": "file"
  },
  {
    "path": "org.go",
    "kind": "file"
  },
  {
    "path": "org_test.go",
    "kind": "file"
  },
  {
    "path": "pr.go",
    "kind": "file"
  },
  {
    "path": "pr_test.go",
    "kind": "file"
  },
  {
    "path": "repo.go",
    "kind": "file"
  },
  {
    "path": "repo_test.go",
    "kind": "file"
  },
  {
    "path": "review.go",
    "kind": "file"
  },
  {
    "path": "review_test.go",
    "kind": "file"
  },
  {
    "path": "stash.go",
    "kind": "file"
  },
  {
    "path": "stash_test.go",
    "kind": "file"
  },
  {
    "path": "testdata/branch.json",
    "kind": "file"
  },
  {
    "path": "testdata/branch.json.golden",
    "kind": "file"
  },
  {
    "path": "testdata/branches.json",
    "kind": "file"
  },
  {
    "path": "testdata/branches.json.golden",
    "kind": "file"
  },
  {
    "path": "testdata/changes.json",
    "kind": "file"
  },
  {
    "path": "testdata/changes.json.golden",
    "kind": "file"
  },
  {
    "path": "testdata/commit.json",
    "kind": "file"
  }
]
